change/office-ui-fabric-react-2020-02-17-11-21-10-master.json

```
@@ -0,0 +1,9 @@
+{
+  "type": "patch",
```

```suggestion
  "type": "minor",
```

Since we're adding to the public API.